types/budget.d.ts

```diff
@@ -40,6 +42,15 @@ declare global {
         budget: number;
       }
 
+      export interface Options {
+        /**
+         * Origins that should be considered first-party.
```

Comment:
this goes a little more to how it will be used, but is this like `extraFirstPartyOrigins` or will the origin of the main resource not be considered first party if it's missing from this array? I guess I could see that usecase with things like wix/squarespace where maybe the root origin isn't actually yours and you load in your first party stuff off a CDN or something? but that seems like a rare user who would do that *and* know how to configure this 😆